backend/src/main/java/sonia/meetapp/app/participants/NewParticipant.java

```diff
@@ -0,0 +1,15 @@
+package sonia.meetapp.app.participants;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class NewParticipant {
+    String name;
+
+    public Participant withRandomID(NewParticipant newparticipant) {
+        return new Participant(newparticipant.name, UUID.randomUUID().toString());
```

Comment:
Ich würde überlegen die UUID in eine eigene IdUtils Klasse auszulagern, dann kannst du im test die Methode leicht mocken...